### DISCUSSION TITLE
How to correspond to the version of docker cri-dockerd k8s?

### DISCUSSION THREAD

**Discussion by liubin95** [Mirantis/cri-dockerd#400] [Q&A]
I need to use k8s 1.30?
Should I use the latest version of cri-dockerd and docker?

**Comment by afbjorklund** [comment]
You _should_ be able to use the latest version.

Eventually it might be nice with a compat table?

i.e. similar to this table, for the containerd runtime:

https://containerd.io/releases/#kubernetes-support

---

It seems to be a common question, for Kubernetes on Docker:

* https://github.com/Mirantis/cri-dockerd/discussions/318
